components/form/Textarea.js

```diff
@@ -14,4 +16,6 @@ export default function Textarea({ name, value, label, ...restProps }) {
       />
     </div>
   );
-}
+})
+
```

Comment:
```suggestion
Textarea.displayName="Textarea"
```
This will resolve the build error